packages/vector_graphics_compiler/lib/src/svg/node.dart

```diff
@@ -147,6 +147,7 @@ class ParentNode extends Node {
   void build(DrawCommandBuilder builder, AffineMatrix transform) {
     final bool requiresLayer = _requiresSaveLayer();
     if (requiresLayer) {
+      // TODO: bounds
```

Comment:
I think we can remove this. If you dn't specify a layer, Skia currently just uses the current clip, which is as good as or better than whatever we'd calculate here.

Reply:
done